pages/Mining.qml

```diff
@@ -106,27 +106,59 @@ Rectangle {
                 Layout.fillWidth: true
                 spacing: 16
 
-                MoneroComponents.LineEdit {
-                    id: soloMinerThreadsLine
-                    Layout.minimumWidth: 200
-                    text: "1"
-                    validator: IntValidator { bottom: 1; top: idealThreadCount }
-                }
+                RowLayout {
+                    MoneroComponents.StandardButton {
+                        id: removeThreadButton
+                        small: true
+                        primary: false
+                        text: qsTr("-") + translationManager.emptyString
```

Comment:
```suggestion
                        text: "−"
```

No need to translate this. Also note that I used a `−` instead of `-` so please copy it from here, not sure if it makes a difference in the program.